src/fsharp/FSharp.Core/FSharp.Core.fsproj

```diff
@@ -222,6 +222,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="System.ValueTuple" Version="4.4.0"  />
```

Comment:
I got compiler errors when not referencing it, so I simply added it, we could certainly use KeyValuePair or some other struct for the implementation but I saw several other suggestions that would require struct-tuples so I didn't go through n removing it but I can certainly do so if you think it's the way to go.

Reply:
The compile errors is surprising ... reference it, if you need to.  I will take a look at why we don't get it vie netstandard2.0

Reply:
Within the iDE (VS) I don't get compile-time errors when using a struct tuple, but maybe something is wrong in the build process?

Reply:
I will take a look, sometime.

Reply:
Hey, this may have been a false alarm, i accidentally pulled `master` first instead of `main`, then switched and cherry-picked my map changes, so it might work without the reference after all.
I can look into that tomorrow (european tomorrow that is 😆)

Reply:
Okay it really was useless, so I removed it again (would have been necessary for the net45 build)